### PR TITLE
Feature/2400/update tos

### PR DESCRIPTION
**Pull request by Richard-Hansen**
Partner PR: https://github.com/dockstore/dockstore/pull/4037
Wiki Page:https://wiki.oicr.on.ca/pages/resumedraft.action?draftId=154540096&draftShareId=4b40e098-bed1-4656-ae25-6a1af482afa6&
Original Ticket: https://ucsc-cgl.atlassian.net/browse/SEAB-2400

### CODE REVIEW

File: src/app/shared/constants.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const servicesURLSegment = 'services';
 export const toolsURLSegment = 'tools';
 export const altAvatarImg = 'http://www.imcslc.ca/imc/includes/themes/imc/images/layout/img_placeholder_avatar.jpg';
 export const devMode = false;
-export const currentTOSVersion: User.TosversionEnum = User.TosversionEnum.TOSVERSION1;
```
Comment (denis-yuen) [inline src/app/shared/constants.ts:36]: A little confused but cannot comment on it directly. 
`src/assets/docs/Dockstore_Terms_of_Service.pdf` shows as a deletion but I don't see an addition.

Comment (Richard-Hansen) [reply src/app/shared/constants.ts:36]: You can view the file by clicking the three dots `...` on the right and hitting `view file`. That will open up the new TOS pdf.

Not sure why we can't view it normally though...

Comment (denis-yuen) [reply src/app/shared/constants.ts:36]: Oh, never mind. Its (just) 81% smaller for some reason, its not fully deleted.

**Review by denis-yuen — APPROVED** [review]

**Review by NatalieEO — APPROVED** [review]

**Review by garyluu — APPROVED** [review]

**Review by coverbeck — COMMENTED** [review]
You're going to have to point at the JARs generated by CircleCI on the hotfix branch for the build to pass. Ultimately we'll need to do another PR where it points to 1.10.2 web service, but that doesn't exist yet.

**Review by coverbeck — APPROVED** [review]

**Pull request merged**